src/Components/SurveyComponent.js

```diff
@@ -2,15 +2,23 @@ import React from 'react';
 import * as Survey from 'survey-react';
 import SURVEY_DATA from '../data/survey.js';
 import { useState, useEffect, useRef } from 'react';
+import useLocalStorage from '../hooks/useLocalStorage'
 
 function SurveyComponent() {
+
+    const [surveyData, setSurveyData] = useLocalStorage(Object.keys, Object.values)
```

Comment:
It looks like the hook expects the local storage key (a string) and its initial value as arguments, so maybe we could pass something like `useLocalStorage("surveyData", null)`. Hope that makes sense, but let me know if I'm missing something

Reply:
That is right. I was thinking about it as if we wanted to call each individual question but we just need the survey data object. Good thinking, thanks.